package.json

```diff
@@ -326,6 +326,7 @@
     "@typescript-eslint/eslint-plugin": "^5.10.1",
     "@typescript-eslint/parser": "^5.10.1",
     "ansi_up": "^5.1.0",
+    "mock-http": "^1.1.0",
```

Comment:
Why do we need this package / extra dependency? Can we live without it?

Reply:
At the moment, no.

> Get rid of IncomingMessage and http.ServerResponse interface from router so that new routes can be unit tested easily even when using router in tests.

However, we can after this.